src/main/java/duke/util/Ui.java

```diff
@@ -0,0 +1,123 @@
+package duke.util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import duke.exception.DukeException;
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * Ui handles everything that is shown to user
+ *
+ * @author Guo-KeCheng
+ */
+public class Ui {
```

Comment:
Good way of laying out the UI, do you know how to incorporate this into level 10?